### PR TITLE
Updated styling in installation guide

### PR DESCRIPTION
**Pull request by satori-ytolstoguzov**
Fixed minor styling issues per discussion in #1862

### CODE REVIEW

**Comment by alvelig** [comment]
May I ask you please to add the following lines:

After `pod install`

**Please make sure you open `.xcworkspace`, not `.xcproject`**

And somewhere there:
**Please make sure, that your XCode is 9.+, otherwise you'll get `use of undeclared identifier 'MKMapTypeMutedStandard'` error**

**Comment by satori-ytolstoguzov** [comment]
@alvelig added. I am not a native english speaker, so feel free to point out if something is incorrect or hard to understand.

@rborn I see that although library is called GoogleMaps, they use "Google Maps" almost everywhere on their site. Let me know if I meant something different.

**Comment by alvelig** [comment]
@satori-ytolstoguzov Neither am I lol. LGTM

File: docs/installation.md [hunk 1]
```diff
@@ -170,6 +177,8 @@ If you have ran 'react-native link` by mistake:
 6. delete ios/build folder
 7. start again with the installation steps
 
+If you use Xcode with version less than 9 you may get `use of undeclared identifier 'MKMapTypeMutedStandard'` error. In this case you have to update your Xcode.
```
Comment (alvelig) [inline docs/installation.md]: Here also `Entry, ":CFBundleIdentifier", Does Not Exist` may happen if not updated to XCode 9. 🙏

Comment (rborn) [reply docs/installation.md]: @alvelig  grrr, merged while you were commenting :(

Comment (rborn) [reply docs/installation.md]: @satori-ytolstoguzov if you want to update this I'll merge it, sorry for the mess 🍺

**Comment by satori-ytolstoguzov** [comment]
@rborn #1914

**Pull request merged**